runtime/testnet/src/config/xcm.rs

```diff
@@ -28,7 +28,8 @@ use xcm_executor::XcmExecutor;
 parameter_types! {
 	pub const RelayLocation: Location = Location::parent();
 	pub AssetHub: Location = Location::new(1, [Parachain(1000)]);
-	pub const RelayNetwork: Option<NetworkId> = None;
+	// 0x77afd6190f1554ad45fd0d31aee62aacc33c6db0ea801129acb813f913e0764f from https://dotapps-io.ipns.dweb.link/?rpc=wss%3A%2F%2Frpc.ibp.network%2Fpaseo#/explorer/query/0
+	pub const RelayNetwork: Option<NetworkId> = Some(ByGenesis([119,175,214,25,15,21,84,173,69,253,13,49,174,230,42,172,195,60,109,176,234,128,17,41,172,184,19,249,19,224,118,79]));
```

Comment:
I think this will be problematic when testing locally, as the genesis block hash will differ when launched via zombienet.

Reply:
Is this a blocker?

Reply:
Not sure. Assume it would only be an issue when using xcm on a local testnet. Emulated tests are not affected.

@al3mart also mentioned that there is a risk that a regenesis of Paseo breaks this, even tho it appears to be the most accurate way. Having said that, the other available enums presumably work on name, so maybe sticking with `Polkadot` is better.

Reply:
Paseo should have not a regenesis, but if if this should be the case then it would need changing.
Given that Paseo is using [`NetworkId::Polkadot`](https://github.com/paseo-network/runtimes/blob/2fa66a19eef96bd2c079385057fab2564d09b751/relay/paseo/src/xcm_config.rs#L56) we should be good doing the change.

Reply:
Updated to `Polkadot`